### ISSUE TITLE
[documentation] constraint.pointBVector wrongly states it is offeset from bodyA

### ISSUE THREAD

**Issue by buffos** [liabru/matter-js#624]
When reading documentation about constraint pointB

```
constraint.pointBVector
A Vector that specifies the offset of the constraint from center of the constraint.bodyA if defined, otherwise a world-space position.
```

you get the impression we are offsetting from bodyA origin

When looking at the code you clearly see that we are offesetting from body B

`initialPointB = constraint.bodyB ? Vector.add(constraint.bodyB.position, constraint.pointB) : constraint.pointB,`

**Comment by Qriva** [comment]
#588 I think it's duplicate.
Yes `pointA `for `bodyA `and `pointB `to `bodyB`

**Comment by liabru** [comment]
Yeah there's an issue with the docs here that needs to be fixed, this is a duplicate so refer to #588, thanks for raising it though.
